library/default-synths-extra.scd

```diff
@@ -578,7 +578,7 @@ SynthDef(\soshats, {
 
   initialEnv = Env.perc(attackTime: 0.005, releaseTime: sustain / 2, curve: -4).kr;
   initialSweep = Line.kr(start: 15000, end: 9000, dur: sustain / 2);
-  initial = BPF.ar(in: root, freq: initialSweep, rq: voice, mul: initialEnv);
+  initial = BPF.ar(in: root, freq: initialSweep, rq: resonance, mul: initialEnv);
```

Comment:
could you please add a default?

Reply:
what do you mean by `default`? `resonance` value is `1` by default, described at the `arg` section in `soshats` synthdef